File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/InfraSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class InfraSearch extends AbstractEntity {
	String itemId;
	String itemName;
	String quantity;
	@Temporal(TemporalType.DATE)
	Date itemAcquisitionDate;
}

File: scholar-web/src/main/webapp/app/model/administration/inventory/infra/InfraSearch.js
Ext.define('scholar.model.administration.inventory.infra.InfraSearch', {
	extend: 'Ext.data.Model',		
	fields : [{
		name : 'id',
		type : 'int'
	}, {
		name : 'itemId',
		type : 'string'
	}, {
		name : 'itemName',
		type : 'string'
	},
	{
		name : 'quantity',
		type : 'string'
	},
	{
		name : 'itemAcquisitionDate',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ]
});